README.markdown

```diff
@@ -314,6 +314,13 @@ or
 
 See [more_set_input_headers](#more_set_input_headers) for more details.
 
+Wildcard `*` can also be used to specify a header name pattern. For example, the following directive effectively clears *any* input headers starting by "`X-Hidden-`":
```

Comment:
@ghedo Yeah, the `more_clear_headers` docs need fixing as well. I just fixed it in master.